Lib/test/test_unicode.py

```diff
@@ -2830,6 +2830,25 @@ def test_asucs4(self):
             self.assertEqual(unicode_asucs4(s, len(s), True), s+'\0')
             self.assertEqual(unicode_asucs4(s, len(s), False), s+'\uffff')
 
+    # Test PyUnicode_GetUTF8Buffer()
+    @support.cpython_only
+    def test_getutf8buffer(self):
+        from _testcapi import unicode_getutf8buffer
+
+        ascii_ = "foo"
```

Comment:
Why not inline these variables?

Reply:
* Follow other tests
* Variable name is a hint why this value is tested.